Pickers/SWActionSheet.m

```diff
@@ -213,11 +223,14 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)interface
     return NO;
 }
 
-// iOS6 support
-// ---
 - (BOOL)shouldAutorotate
 {
-    return YES;
+    if (self.maskVC == UIInterfaceOrientationPortrait){
```

Comment:
it could be realised easily

Reply:
What do you mean? Can you please elaborate or suggest a better approach?

Reply:
Please, put it this way, so it makes the style more readable. it's just about good code style